st2tests/st2tests/base.py

```diff
@@ -599,7 +599,7 @@ def _stop_running_processes(self):
                 except:
                     stderr = None
 
-                print("Stopping process with pid %s" % (process.id))
+                print("Stopping process with pid %s" % (process.pid))
```

Comment:
Extract this one character bugfix into its own PR. Not sure why we aren't hitting this more often than we are.

Reply:
#6184